docs/node/statesync.mdx

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 5
+sidebar_label: "Statesync"
+id: "Statesync"
+title: "Using State Sync"
+description: "Learn how to bootstrap a new node using statesync"
+slug: /node/statesync
+---
+
+A new node joining the network needs to catchup with the existing nodes to be able to participate in the network. The default approach the node uses to catch up with the network is to replay all the blocks from the genesis. This process can take a long time depending on the number of blocks in the chain. Statesync is a feature that allows a new node to bootstrap directly from a snapshot instead of replaying all the blocks from the genesis. This rapidly reduces the time it takes to sync a new node to the network, but the node will have a truncated history starting from the height of the snapshot.
+
+:::note
+Statesync is not attempted on a partially synced node. If you have a partially synced node, it uses blocksync by default to sync the remaining blocks.
+:::
+
+## Trusted Snapshot Providers
+
+To support statesync, each network should have atleast two trusted snapshot providers that are responsible for creating, distributing and validating snapshots. These trusted snapshot providers should have [snapshot creation](/docs/daemon/config/settings#appsnapshots) enabled and these provider's chain P2P and RPC endpoints should be accessible to the other nodes in the network to respond to the snapshot discovery and validation requests from the new nodes.
```

Comment:
Is it possible to just run with 1? If they are trusted anyways, I don't see why 2 is necessary. I would imagine most users would just run two machines, which would seem redundant.

Reply:
umm, actually it's a constraint from comet, it won't start statesync if there are less than 2 providers.